[rospy.client][INFO] 2022-01-07 16:27:32,760: init_node, name[/brain_14665_1641553052724], pid[14665]
[xmlrpc][INFO] 2022-01-07 16:27:32,760: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2022-01-07 16:27:32,761: Started XML-RPC server [http://prajwal-IdeaPad-L340-15IRH-Gaming:39287/]
[rospy.init][INFO] 2022-01-07 16:27:32,761: ROS Slave URI: [http://prajwal-IdeaPad-L340-15IRH-Gaming:39287/]
[rospy.impl.masterslave][INFO] 2022-01-07 16:27:32,761: _ready: http://prajwal-IdeaPad-L340-15IRH-Gaming:39287/
[rospy.registration][INFO] 2022-01-07 16:27:32,762: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2022-01-07 16:27:32,762: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2022-01-07 16:27:32,861: registered with master
[rospy.rosout][INFO] 2022-01-07 16:27:32,861: initializing /rosout core topic
[rospy.rosout][INFO] 2022-01-07 16:27:32,864: connected to core topic /rosout
[rospy.simtime][INFO] 2022-01-07 16:27:32,865: initializing /clock core topic
[rospy.simtime][INFO] 2022-01-07 16:27:32,867: connected to core topic /clock
[rospy.internal][INFO] 2022-01-07 16:27:32,870: topic[/clock] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:39413/], count 0
[rospy.internal][INFO] 2022-01-07 16:27:32,876: topic[/grid3/allRobots/poses] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:43429/], count 0
[rospy.internal][INFO] 2022-01-07 16:27:32,880: topic[/grid3/planner/status] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:37255/], count 0
[rospy.internal][INFO] 2022-01-07 16:27:32,895: topic[/grid3/allRobots/goals] adding connection to [/trajectory_planner], count 0
[rospy.internal][INFO] 2022-01-07 16:27:33,070: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2022-01-07 16:27:33,874: topic[/grid3/droppingCompleted] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/], count 0
[rospy.internal][INFO] 2022-01-07 16:27:33,879: topic[/grid3/packageLoaded] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/], count 0
[rospy.internal][ERROR] 2022-01-07 16:34:08,058: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:46595 (http://prajwal-IdeaPad-L340-15IRH-Gaming:37255/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2022-01-07 16:34:08,058: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:38537 (http://prajwal-IdeaPad-L340-15IRH-Gaming:43429/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2022-01-07 16:34:08,060: topic[/grid3/allRobots/goals] adding connection to [/trajectory_planner], count 1
[rospy.internal][INFO] 2022-01-07 16:34:08,061: topic[/grid3/allRobots/goals] removing connection to /trajectory_planner
[rospy.internal][ERROR] 2022-01-07 16:34:08,099: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:40487 (http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: [/rqt_gui] is not a publisher of [/grid3/droppingCompleted]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/grid3/packageLoaded', 'grid3/packageLoaded'], ['/robot1/cmd_vel', 'geometry_msgs/Twist'], ['/robot2/cmd_vel', 'geometry_msgs/Twist'], ['/robot3/cmd_vel', 'geometry_msgs/Twist'], ['/robot4/cmd_ve', 'geometry_msgs/Twist']]

[rospy.internal][ERROR] 2022-01-07 16:34:08,102: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:40487 (http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: [/rqt_gui] is not a publisher of [/grid3/packageLoaded]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/robot1/cmd_vel', 'geometry_msgs/Twist'], ['/robot2/cmd_vel', 'geometry_msgs/Twist'], ['/robot3/cmd_vel', 'geometry_msgs/Twist'], ['/robot4/cmd_ve', 'geometry_msgs/Twist']]

[rospy.internal][INFO] 2022-01-07 16:34:08,481: topic[/grid3/allRobots/goals] removing connection to /trajectory_planner
[rospy.internal][WARNING] 2022-01-07 16:34:09,060: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:46595 (http://prajwal-IdeaPad-L340-15IRH-Gaming:37255/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 560, in connect
    self.socket.connect((dest_addr, dest_port))
ConnectionRefusedError: [Errno 111] Connection refused

[rospy.internal][INFO] 2022-01-07 16:34:09,060: topic[/grid3/planner/status] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:37255/
[rospy.internal][WARNING] 2022-01-07 16:34:09,061: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:38537 (http://prajwal-IdeaPad-L340-15IRH-Gaming:43429/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rosgraph/network.py", line 357, in read_ros_handshake_header
    d = sock.recv(buff_size)
ConnectionResetError: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2022-01-07 16:34:09,061: topic[/grid3/allRobots/poses] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:43429/
[rospy.internal][WARNING] 2022-01-07 16:34:09,101: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:40487 (http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 560, in connect
    self.socket.connect((dest_addr, dest_port))
ConnectionRefusedError: [Errno 111] Connection refused

[rospy.internal][INFO] 2022-01-07 16:34:09,101: topic[/grid3/droppingCompleted] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/
[rospy.internal][WARNING] 2022-01-07 16:34:09,103: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:40487 (http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 560, in connect
    self.socket.connect((dest_addr, dest_port))
ConnectionRefusedError: [Errno 111] Connection refused

[rospy.internal][INFO] 2022-01-07 16:34:09,103: topic[/grid3/packageLoaded] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:34139/
[rospy.internal][ERROR] 2022-01-07 16:34:10,009: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:37395 (http://prajwal-IdeaPad-L340-15IRH-Gaming:39413/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 37395 to [127.0.0.1:57598 on socket 17]] [/brain_14665_1641553052724].

[rospy.internal][WARNING] 2022-01-07 16:34:11,010: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:37395 (http://prajwal-IdeaPad-L340-15IRH-Gaming:39413/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 560, in connect
    self.socket.connect((dest_addr, dest_port))
ConnectionRefusedError: [Errno 111] Connection refused

[rospy.internal][INFO] 2022-01-07 16:34:11,010: topic[/clock] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:39413/
[rospy.internal][INFO] 2022-01-07 16:34:25,490: topic[/rosout] removing connection to /rosout
